methods/lcarun.ts

```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
 
 import { Lci } from 'models/schema';
 import { RunParams } from './lca.model';
-// import { listenerCount } from 'cluster';
+
+// NonrenewableUnitParams were used to change units for nonrenewable group. Units change is not needed for other groups
+interface NonrenewableUnitParams {
+  'brownCoal': number;
+  'hardCoal': number;
+  'crudeOil': number;
+  'mineGas': number;
+  'naturalGas': number;
+  'Uranium': number;
+  [key: string]: number;
+}
+const unitParams: NonrenewableUnitParams = { 'brownCoal': 9.9, 'hardCoal': 19.1, 'crudeOil': 45.8,
+                                             'mineGas': 32.45, 'naturalGas': 32.12, 'Uranium': 560000};
 
 const lcarun = async (params: RunParams, db: knex) => {
-  const rows: Lci[] = await db
+  const inputRows: Lci[] = await db
     .table('lci_input')
-    .where({ lci_group: 'renewable' });
+    .where({ lci_group: 'renewable' }).orWhere({ lci_group: 'nonrenewable' }).orWhere({ lci_group: 'water' });
+
+  const outputRows: Lci[] = await db
```

Comment:
the naming here is kind of confusing

Reply:
renamed in new branch KL/lci_claculation